mythril/analysis/solver.py

```diff
@@ -83,19 +90,19 @@ def get_transaction_sequence(global_state, constraints):
 
     transaction_sequence = global_state.world_state.transaction_sequence
 
-    # gaslimit & gasprice don't exist yet
     tx_template = {
```

Comment:
can we remove this initialization. There is only one use, so it does not make much sense to initialize this here